source/define-const-property.js

```diff
@@ -0,0 +1,23 @@
+'use strict';
+const is = require('@sindresorhus/is');
+
+const deepFreeze = obj => {
+	for (const [key, value] of Object.entries(obj)) {
+		if (is.object(value)) {
+			deepFreeze(obj[key]);
+		}
+	}
+
+	return Object.freeze(obj);
+};
+
+module.exports = (where, properties, options = {}) => {
+	for (const [key, value] of Object.entries(properties)) {
+		Object.defineProperty(where, key, {
+			value: options.deepFreeze ? deepFreeze(value) : Object.freeze(value),
+			writable: false,
+			enumerable: true,
+			configurable: true
```

Comment:
I see it was like this before, but why is `configurable` set to `true`? We're locking it unless you're serious enough about changing it to redefine the property?

Reply:
Yeah. It should be set to `false` (default value).